relational/table.py

```diff
@@ -116,24 +160,55 @@ def _gen_facts(self, row, proj):
             row[self.index(self.PERMIT_AUTHORITY)] = PPTS.OUTPUT_NAME
             row[self.index(self.PERMIT_AUTHORITY_ID)] = proj.field(
                 'fk', PPTS.NAME)
+        elif proj.field('project_id', MOHCDPipeline.NAME) != '':
+            row[self.index(self.ADDRESS)] = '%s %s, %s' % (
+                    proj.field('street_name', MOHCDPipeline.NAME),
```

Comment:
Hmm I thought this would also need street_number? (here and for inclusionary)

Reply:
ah, the two records i looked at had no street_number so i didn't think it existed. my bad.